coffeine/pipelines.py

```diff
@@ -282,6 +301,16 @@ def _get_projector_vectorizer(projection, vectorization,  kernel=None):
     elif method == 'riemann_wasserstein':
         steps = (ProjIdentitySpace, RiemannSnp)
 
+    # add alignment options
+    alignment_steps = []
+    if alignment is None:
+        alignment_steps = None
+    else:
+        if 're-center' in alignment:
+            alignment_steps.append(ReCenter(domains=domains))
+        if 're-scale' in alignment:
+            alignment_steps.append(ReScale(domains=domains))
```

Comment:
See comment above. We would use the same ReCenter/ReScale instances for every column of the data frame based on this code here, right?

Reply:
I don't understand, is it the same thing as what is done a few lines above for the projection and vectorization steps?
Also in practice I checked and each frequency band is aligned independently.

Reply:
It’s not the same as we instantiate the classes independently for every column. We can check of course if this is practically unneccessary by now because of internal cloning etc. But if you want to be consistent with the tested / save code you can just adopt the same pattern as for the other projection/vectorization steps. You currently instantiate the classes globally and plug in the same input column by column.

Reply:
Ok I see, let's do it as projection/vectorization.